rudder-web/src/main/scala/com/normation/rudder/web/components/NodeGroupForm.scala

```diff
@@ -177,7 +177,7 @@ class NodeGroupForm(
       "rudderid" -> <div><b class="threeCol">Rudder ID: </b>{nodeGroup.id.value}</div>,
       "cfeclasses" -> <div>
 											  <a href="#" onclick={s"$$('#cfe-${nodeGroup.id.value}').toggle(300); return false;"}>
-												<b class="threeCol">Display CFEngine classes</b>
+												<b class="threeCol">Display classes generated for this group</b>
```

Comment:
Love to have a shorter / lighter thing.

Why not for "conditional". (but maybe people will not understand where it can be used)

My idea; "display agent classes", what do you think of it ?